styles/style.css

```diff
@@ -0,0 +1,12 @@
+header{
+    font-family: 'Roboto', sans-serif;
+     color:#c4c4c4;
+    font-size: 200px;
```

Comment:
At least for font-size, margin and paddig, we should use rem instead of px. Can you convert this? 1rem == 16px, but you can use this calculator here as well https://nekocalc.com/px-to-rem-converter